wallabag/wallabag
# Add basic docker-compose configuration

### What's it ?

A basic and docker-based environment for developing wallabag.

### What's included

* nginx
* PHP7 FPM 

### What's missing

* probably some php extensions (xdebug ? others ?)
* proper unix right management, keep at hand
```
sudo chown www-data -R app/cache app/logs app/bootstrap.php.cache data/db
```
* a docker-compliant firewall for `app-dev.php`

### How to test

1. make sure docker and docker-compose are installed and up-to-date
2. clone project and `composer install` it
3. `docker-compose up`
4. browse to [http://localhost:8080](http://localhost:8080/), tune rights and cache generation
